kubernetes-manifests/emailservice.yaml

```diff
@@ -26,6 +26,10 @@ spec:
         app: emailservice
     spec:
       terminationGracePeriodSeconds: 5
+      initContainers:
```

Comment:
add a comment to explain why you do this (also applies to other yaml files where you have this)

Reply:
Added! Thanks for feedback